src/useKeyboardEvent/useKeyboardEventStories.test.ts

```diff
@@ -0,0 +1,39 @@
+import '@testing-library/jest-dom';
+import { waitFor, render } from '@muban/testing-library';
+import { Key } from 'ts-key-enum';
+import { Demo } from './useKeyboardEvent.stories';
+
+describe('useKeyboardEvent stories', () => {
+  it('should render', () => {
+    const { getByText } = render(Demo);
+
+    expect(getByText('Test Area')).toBeInTheDocument();
+  });
+
+  it('should press the `enter` key', async () => {
+    const { getByText } = render(Demo);
+
+    document.dispatchEvent(new KeyboardEvent('keydown', { key: Key.Enter }));
```

Comment:
`testing-library` has helpers to make this easier: https://testing-library.com/docs/user-event/keyboard.
Not sure, but it might even respect the document focus.